plotly/plotly_aux/getuserdir.m

```diff
@@ -1,13 +1,13 @@
-function userDir = getuserdir
-% GETUSERDIR  Retrieve the user directory
-%   - Under Windows returns the %APPDATA% directory
-%   - For other OSs uses java to retrieve the user.home directory
-
-if ispc
-    %     userDir = winqueryreg('HKEY_CURRENT_USER',...
-    %         ['Software\Microsoft\Windows\CurrentVersion\' ...
-    %          'Explorer\Shell Folders'],'Personal');
-    userDir = getenv('appdata');
-else
-    userDir = char(java.lang.System.getProperty('user.home'));
+function userDir = getuserdir
+% GETUSERDIR  Retrieve the user directory
+%   - Under Windows returns the %APPDATA% directory
+%   - For other OSs uses java to retrieve the user.home directory
+
+if ispc
+    %     userDir = winqueryreg('HKEY_CURRENT_USER',...
```

Comment:
Do we need to keep these comments? I don't understand if they are helpful?

Reply:
which one ?